app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
+
+  .pagination {
+    margin-bottom: $small-spacing;
+  }
+
+  .pagination-total {
+    text-align: center;
```

Comment:
Is this needed? The `.pagination` on Line 1 already sets this, and it is the parent to `. pagination-total`. Seems like we should be able to remove this class all-together.